packages/kolibri-common/composables/useCoachMetadataTags.js

```diff
@@ -0,0 +1,74 @@
+import { ref } from 'kolibri.lib.vueCompositionApi';
+import { ContentNodeKinds } from 'kolibri.coreVue.vuex.constants';
+import coreStrings from 'kolibri.utils.coreStrings';
+import commonCoreStringsMixin from 'kolibri.coreVue.mixins.commonCoreStrings';
+import { learnStrings } from '../../../kolibri/plugins/learn/assets/src/views/commonLearnStrings';
```

Comment:
We don't want to import across plugins. I do realize that there was a previous comment here about whether or not we had a 'Multiple learning activities' string from previous work, and to use it if we do have it, which is what this is trying to address. But we still don't want to import it cross plugin from Learn. So instead, let's move the string into `commonCoreStrings`, and then update all of the uses to use that (both here, and any places it's used in Learn)